3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -1,35 +1,55 @@
-'use strict';
-/*------------------------------------------------------------------------------
-Complete the four functions provided in the starter `index.js` file:
-
-`fetchData`: In the `fetchData` function, make use of `fetch` and its Promise 
-  syntax in order to get the data from the public API. Errors (HTTP or network 
-  errors) should be logged to the console.
-
-`fetchAndPopulatePokemons`: Use `fetchData()` to load the pokemon data from the 
-  public API and populates the `<select>` element in the DOM.
-  
-`fetchImage`: Use `fetchData()` to fetch the selected image and update the 
-  `<img>` element in the DOM.
-
-`main`: The `main` function orchestrates the other functions. The `main` 
-  function should be executed when the window has finished loading.
-
-Try and avoid using global variables. Instead, use function parameters and 
-return values to pass data back and forth.
-------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
+const buttonGetPokemon = document.createElement('button');
+document.body.appendChild(buttonGetPokemon);
+
+buttonGetPokemon.setAttribute('type', 'submit');
+buttonGetPokemon.textContent = 'Get Pokemon!';
+
+const form = document.createElement('form');
+document.body.appendChild(form);
+
+const selectPokemon = document.createElement('select');
+form.appendChild(selectPokemon);
+
+const pokemonImage = document.createElement('img');
```

Comment:
The assignment asked for you to avoid global variables like the ones you have defined here. Can you move them to inside the function `main()`? It will probably require you to pass some of these variables as arguments to other functions. But it will improve the readability of your code by making more explicit what data each function needs.

Reply:
This comment is still unaddressed.